docs/beyond_basics/101-173-contributing.rst

```diff
@@ -0,0 +1,28 @@
+.. _contributing_datalad:
+
+Contributing to DataLad
+-----------------------
+
+
+DataLad is free and open source software.
+Everyone can contribute in various forms -- feature requests, questions, artwork, tutorials, code patches, bug reports, ... even follows, likes, or retweets on `Twitter <https://twitter.com/datalad>`_, or discussions in our `matrix chatroom <https://app.element.io/#/room/%23datalad:matrix.org>`_.
+We would delighted to hear from you in any form.
+
+.. figure:: ../artwork/src/join.svg
+   :width: 50%
+
+The following resources could be helpful:
+
+Contributing to the handbook
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Take a look at the section :ref:`contribute` for more information.
+
+Contributing to DataLad
+^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Show your support**: If you like DataLad you can show your support in various meaningful ways. You can `"star" <https://github.com/datalad/datalad/stargazers>`_ the project on GitHub. You can follow the `DataLad Twitter Account <https://twitter.com/datalad>`_ and like and retweet our updates. You can cite the `paper about DataLad <https://joss.theoj.org/papers/10.21105/joss.03262>`_ in academic works or blog posts. You can tell your friends, family, and colleagues about it, or come chat with us about what you do on `Matrix <https://app.element.io/#/room/%23datalad:matrix.org>`_ (a free, decentral, and secure communication network). Whatever you do - thank you!
```

Comment:
"following" somewhat reiterates/duplicates the opening paragraph.  Even the name "Contributing to DataLad" chapter/section is duplicated.
I wonder if there is some what to have it only in a single place.  Also worth adding YouTube channel people *should* subscribe to

Reply:
"decentral" ==> might just be my prior exposure but I haven't come across this phrase; perhaps decentralis(z)ed?

Reply:
great points, thanks!